src/linagora.esn.unifiedinbox/app/components/inbox-configuration/forwardings/inbox-forwardings.controller.js

```diff
@@ -2,8 +2,10 @@
 
 const _ = require('lodash');
 
-require('../../../services/forwardings/inbox-forwardings.service.js');
-require('../../../services/forwardings/inbox-forwardings-api-client.service.js');
+//require('../../../services/forwardings/inbox-forwardings.service.js');
+//require('../../../services/forwardings/inbox-forwardings-api-client.service.js');
+require('../../../../../esn.inbox.libs/app/services/forwarding/inbox-forwardings.service.js');
+require('../../../../../esn.inbox.libs/app/services/forwarding/inbox-forwardings-api-client.service.js');
```

Comment:
no need this since you require the angular module `esn.inbox.lib` module file in app.module